src/instreamTracking.js

```diff
@@ -0,0 +1,106 @@
+import { config } from './config.js';
+import { auctionManager } from './auctionManager.js';
+import { VIDEO } from './mediaTypes.js';
+import { OUTSTREAM } from './video.js';
+import * as events from './events.js';
+import * as utils from './utils.js';
+import { TARGETING_KEYS, EVENTS, BID_STATUS } from '../src/constants.json';
+
+const { CACHE_ID, UUID } = TARGETING_KEYS;
+const { BID_WON } = EVENTS;
+const { RENDERED } = BID_STATUS;
+
+const INSTREAM_TRACKING_DEFAULT_CONFIG = {
+  enabled: false,
```

Comment:
I am wondering if this should be enabled to true by default

Reply:
We prefer not enabling it by default as its not the most performant option, also there can be publishers who are not using any prebid analytics or video monetization, so let the publisher set it as a option